.github/workflows/ci.yml

```
@@ -217,15 +217,15 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v4
         with:
-          java-version: '11'
+          java-version: '17'
```

cc @weizhouapache I think keeping the build.yaml to build with JDK11 and the ci.yml with JDK17 would simulate against new env using JRE17 (while code can still be built with JDK11).

---

ok @rohityadavcloud 
let's revert the change

```suggestion
          java-version: '11'
```